generator/internal/language/rusttemplate.go

```diff
@@ -364,20 +401,28 @@ func rustAnnotateOneOf(oneof *api.OneOf, message *api.Message, state *api.APISta
 	}
 }
 
-func rustAnnotateField(field *api.Field, state *api.APIState, modulePath, sourceSpecificationPackageName string, packageMapping map[string]*rustPackage) {
-	if field == nil {
-		return
-	}
-	field.Codec = &rustFieldAnnotations{
+func rustAnnotateField(field *api.Field, message *api.Message, state *api.APIState, modulePath, sourceSpecificationPackageName string, packageMapping map[string]*rustPackage) {
+	ann := &rustFieldAnnotations{
 		FieldName:          rustToSnake(field.Name),
 		SetterName:         rustToSnakeNoMangling(field.Name),
+		FQMessageName:      rustFQMessageName(message, modulePath, sourceSpecificationPackageName, packageMapping),
```

Comment:
We need this to create a good link. Note that nested messages need a more complex name than just `create::model::{{MessageName}}`.